tests/services/test_service__create.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
```

Comment:
moderate: double underscore in the file name. `test_service__create.py` --> `test_service_create.py`

Reply:
yeah.. because we are testing `_create` and not `create` :sweat_smile: